src/Symfony/Component/Validator/Constraints/BicValidator.php

```diff
@@ -69,13 +68,15 @@ public function validate($value, Constraint $constraint)
             return;
         }
 
+        // @deprecated since Symfony 4.2, will throw a LogicException in 5.0
         if (!class_exists(Intl::class)) {
-            throw new LogicException('The "symfony/intl" component is required to use the Bic constraint.');
+            @trigger_error('Using the Bic constraint without the "symfony/intl" component installed is deprecated since Symfony 4.2.', E_USER_DEPRECATED);
```

Comment:
I would trigger the deprecation in the constraint instead so that developers notice it without having to actually validate anything (that may go without notice if you don't have tests covering it).

Reply:
done